es-app/src/Gamelist.cpp

```diff
@@ -234,7 +234,7 @@ void updateGamelist(SystemData* system)
 					continue;
 				}
 
-				std::string nodePath = Utils::FileSystem::getCanonicalPath(Utils::FileSystem::resolveRelativePath(pathNode.text().get(), system->getStartPath(), true));
+				std::string nodePath = Utils::FileSystem::getCanonicalPath(Utils::FileSystem::resolveRelativePath(pathNode.text().get(), system->getStartPath(), true, false));
```

Comment:
Same here - this is on save. Why are we removing this check here?

Reply:
Same remark.